x-pack/qa/sql/src/main/resources/command.csv-spec

```diff
@@ -174,6 +174,14 @@ test_emp_copy      |BASE TABLE
 test_emp_with_nulls|BASE TABLE
 ;
 
+showTablesIdentifierPatternOnAliases
+SHOW TABLES "test*,-test_emp*";
+
+ name:s        | type:s
+test_alias     |ALIAS          
```

Comment:
I see how this one *should* show aliases. Can you add a test that *shouldn't* show aliases so that it hits more of the change you made above?

Reply:
Unfortunately that doesn't currently work; I've raised a bug for it: https://github.com/elastic/elasticsearch/issues/33395